python/ray/tests/test_autoscaler.py

```diff
@@ -142,45 +142,56 @@ def terminate_node(self, node_id):
 class LoadMetricsTest(unittest.TestCase):
     def testUpdate(self):
         lm = LoadMetrics()
-        lm.update("1.1.1.1", {"CPU": 2}, {"CPU": 1})
+        lm.update("1.1.1.1", {"CPU": 2}, {"CPU": 1}, {})
```

Comment:
Add some test cases for the new metric

Reply:
Yeah there are a couple entries below in testLoadMessages